source/tutorials/nvidia.rst

```diff
@@ -26,17 +26,37 @@ require a manual installation.
    be required.
 
 .. contents:: :local:
-    :depth: 1
+    :depth: 2
 
 Prerequisites
 *************
 
 * A |CL| system with a desktop installed
 * An NVIDIA device installed
 
-Install DKMS
+.. note:: NVIDIA Optimus
+
+
+   Systems with hybrid graphics commonly found on laptops, known as `NVIDIA
```

Comment:
Revise to:  Some systems with hybrid graphics, such as the `NVIDIA Optimus*...` are designed to [...]

Note: Add an asterisk to Optimus.

The logic of the first sentence suggests that all hybrid graphics on laptops use NVIDIA Optimus technology.  I don't think that's what you mean. Please clarify it it is.

Reply:
Yeah, I rewrote these section a lot when drafting this. That is sort of the implication in context of this article. I took another shot at it. Let me know what you think.